packages/advi/advi.2.0.0/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+synopsis: "Active DVI Dune package!"
+description: """
+
+Active-DVI is a DVI previewer and presenter written in Objective Caml
+(http://caml.inria.fr/ocaml/) with some eye candy effects for
+presentation, support for interactive demonstrations, and embedding of
+arbitrary applications within the presentation.
+
+Active-DVI is very well suited for use in combination with WhizzyTeX
+(http://cristal.inria.fr/whizzytex/)
+"""
+maintainer: ["Didier Rémy <Didier.Remy@inria.fr>"]
+authors: [
+  "Alexandre Miquel"
+  "Jun Furuse"
+  "Didier Rémy"
+  "Pierre Weis"
+  "Xavier Leroy"
+  "Roberto Di Cosmo"
+  "Didier Le Botlan"
+  "Alan Schmitt"
+]
+license: "<name>"
+homepage: "http://advi.inria.fr"
+doc: "https://gitlab.inria.fr/remy/advi"
+bug-reports: "Didier.Remy@inria.fr"
+depends: [
+  "ocaml-base-compiler" {>= "4.11.1"}
+  "dune" {>= "2.5"}
+  "graphics" {>= "5.1.1"}
+  "camlimages" {>= "5.0.4"}
+  "conf-texlive"
+]
```

Comment:
```suggestion
  "conf-which"
]
```
Looking more carefully arch failure seems to be related to the use of which in the script